multiqc/modules/samtools/flagstat.py

```diff
@@ -51,7 +51,7 @@ def parse_samtools_flagstats(self):
             keys = OrderedDict()
             reads = {
                 'min': 0,
-                'modify': lambda x: float(x) / 1000000.0,
+                'modify': lambda x: float(x) * config.read_count_multiplier,
                 'suffix': 'M reads',
```

Comment:
```suggestion
                'suffix': '{} reads'.format(config.read_count_prefix),
```

The GitHub suggestion thing isn't working properly for me, but this is what I mean..

Reply:
Ah, it was just the preview that wasn't working..